classes/lable.rb

```diff
@@ -0,0 +1,16 @@
+class Label
+  attr_accessor :title, :color
+  attr_reader :id, :items
+
+  def initialize(id:, title:, color:)
+    @id = id || Random.rand(1..1000)
```

Comment:
@Chimwemwe-127001 I think the id should be automatically given to the label without  giving the user the choice to choose

Reply:
Noted. Working on the changes now.